### DISCUSSION TITLE
MolFromMol2File doesn't read .mol2 file generated by rdkit Mol2Writer.py

### DISCUSSION THREAD

**Discussion by fabioatuib** [rdkit/rdkit#4091] [Q&A]
Hello,

Is it an expected behavior that .mol2 files generated by Mol2Writer.py (part of the RDKit) won't be readable by MolFromMol2File? (is it a release specific problem? I am using 2020.09.1) 
Is there any RDKit solution to write .mol2 files that are readable in RDKit?

Thank you in advance!

[Mol2Writer.py.txt](https://github.com/rdkit/rdkit/files/6398765/Mol2Writer.py.txt)

**Comment by greglandrum** [comment]
I'm not aware of `Mol2Writer.py` file which is actually part of the RDKit. 
There's a file which is part of an old PR #415 which was never merged (because it's not finished), but I don't think there's anything else.
